Added a template for an issue created for a FLIP (#127)
onflow/flips: README.md

1. _(Optional)_ Recruit a sponsor from the maintainers of the project for which your FLIP concerns. A sponsor may help streamline the review process and moderate the discussion with the community.

   Please note that you are not required to recruit a sponsor; this step is entirely optional. If you recruit a sponsor, identify them in the FLIP before posting the PR in step 2.

> 2. Create an [issue](https://github.com/onflow/flips/issues/new/choose) using the FLIP issue template. The title of the issue should be the title of your FLIP e.g. "Dynamic Inclusion fees".

Reviewer: ```suggestion
2. Create an [issue](https://github.com/onflow/flips/issues/new/choose) using the FLIP issue template. The title of the issue should be the title of your FLIP, e.g., "Dynamic Inclusion fees".
```

>    Assign an appropriate label for the type of the FLIP (one of `flip:application`, `flip:governance`, `flip:protocol`, `flip:cadence`)
> 
>    Submit the issue.
> 
>    Note the issue number that gets assigned e.g. for issue https://github.com/onflow/flips/issues/76, the issue number is `76`.

Reviewer: ```suggestion
   Assign an appropriate label for the type of the FLIP (one of `flip:application`, `flip:governance`, `flip:protocol`, `flip:cadence`).
   If you do not have the permission, you may need to ask the sponsor to assist you.

   Submit the issue.

   Note the issue number that gets assigned. For example, for issue https://github.com/onflow/flips/issues/76, the issue number is `76`.
```

> 4. Create your FLIP as a pull request to this repository ([`onflow/flips`](https://github.com/onflow/flips)).

Reviewer 2: In this step can we add a reminder to link the PR(s) to the issue?

Author: yeah - i did mentioned - "Mention the FLIP issue by copying the GitHub URL or the issue in the comment section." lemme know if that will suffice.

   If you have images or other auxiliary files,
   create a directory of the form `YYYYMMDD-descriptive-name` in which to store those files.

   Use the issue number generated in step 2 as the FLIP number.

>    Mention the FLIP issue by copying the github url or the issue in the comment section.

Reviewer: ```suggestion
   Mention the FLIP issue by copying the GitHub URL or the issue in the comment section.
```

   At the top of the PR identify how long the comment period will be.
   This should be a minimum of two weeks from posting the PR.

   
4. Send a message to the #developers channel on [Discord](https://discord.gg/flow)
   with a brief description, a link to the PR and a request for review.

> 5. On the FLIP issue that was created earlier in Step 2, set the status under the "FLIPs Tracker" project to `Proposed`. 

Reviewer: ```suggestion
5. On the FLIP issue that was created earlier in Step 2, set the status under the "FLIPs Tracker" project to `Proposed`. 
   If you do not have the permission, you may need to ask the sponsor to assist you.
```

6. The author may request a review committee meeting after sufficient discussion has taken place.
   This meeting will include the FLIP author, core contributors and interested community members.
   If the discussion is lively, wait until it has settled before going to review.
   The goal of the review meeting is to resolve minor issues;
   consensus should be reached on major issues beforehand.

7. The meeting may approve the FLIP, reject it, or require changes before it can be considered again.
   FLIPs will be merged into this repository ([`onflow/flips`](https://github.com/onflow/flips))
   with the outcome of the review process (approval, rejection).

> 8. Update the status of the FLIP issue under the "FLIPs Tracker" project as per the outcome of step 7 (`Accepted` or `Rejected`)

Reviewer: ```suggestion
8. Update the status of the FLIP issue under the "FLIPs Tracker" project as per the outcome of step 7 (`Accepted` or `Rejected`).
   If you do not have the permission, you may need to ask the sponsor to assist you.
```

## Proposal states

* **Proposed:** The FLIP has been proposed and is awaiting review.
* **Rejected:** The FLIP has been reviewed and rejected.
* **Accepted:** The FLIP has been accepted and is either awaiting implementation or is actively being implemented.
* **Implemented (in VERSION):** The FLIP has been implemented.

## Community members

As the purpose of FLIPs is to ensure the community is well represented and served by new changes to Flow,
it is the responsibility of community members to participate in reviewing FLIPs where they are interested in the outcome.

Community members should:

* Provide feedback as soon as possible to allow adequate time for consideration
* Read FLIPs thoroughly before providing feedback

> * Be civil and constructive (see [Code of Conduct](CODE_OF_CONDUCT.md))

Reviewer 2: The link to the Code of Conduct doesn't open - should be removed

Reviewer: The link should not be removed, instead, the missing Code of Conduct should be added to the repo: https://github.com/onflow/flow/blob/master/CODE_OF_CONDUCT.md

A sponsor is a Flow maintainer, selected by the author at will, and is responsible for ensuring the best possible outcome of the FLIP process.

> In particular, a sponsor may help with the following-

Reviewer: ```suggestion
In particular, a sponsor may help with the following:
```
